Navbar design

Bulatypov opened the pull request:
Changed navbar design.
Navbar becomes a hamburger-menu when width is 500px.

Commit:
Changed navbar design
Navbar becomes a hamburger-menu when width is 500px.

Bulatypov committed:
Update ProfileLink.svelte

Fixed margin for "Log out" button

KekStroke commented on src/lib/components/navigation/NavBar.svelte:
```diff
@@ -1,62 +1,131 @@
-	<img class="logo" src={logo} alt="Innopolis Table Tennis Club" />
+	<input type="checkbox">
+	<div class="hamburger-lines">
+		<span class="line line1"></span>
+		<span class="line line2"></span>
+		<span class="line line3"></span>
```
span со свойством display: block - это тот же div. Поменяй эти span'ы на div'ы

KekStroke commented on src/lib/components/navigation/NavBar.svelte:
```diff
@@ -1,62 +1,131 @@
-	@media (max-width: 800px) {
-		nav {
-			padding: 0 1em;
+	@media (max-width: 500px) {
```
Чем ты руководствовался, когда ставил max-width 500px? Смартфоны разве не 480px?

KekStroke commented on src/lib/components/navigation/ProfileLink.svelte:
```diff
@@ -34,18 +32,53 @@
 	}
-
-	@media (max-width: 450px) {
+	@media (max-width: 500px) {
```
Почему 500px?

KekStroke commented on src/lib/components/navigation/ProfileLink.svelte:
```diff
@@ -34,18 +32,53 @@
 			gap: 0.5em;
 		}
+		.sign-in {
+			margin-top: 1px;
```
Почему в пикселях margin? Посмотри сколько размер шрифта и возьми обратное ему число rem для 1 пикселя. Например если шрифт дефолтного размера (16px), то можно поставить 1/16 = 0.0625 rem

Commit:
Fixed mistakes according to review:
Changed span to div
Changed max-width from 500px to 480px
Changed margin-top from px to rem

Commit:
Fixed mistakes according to review:
Changed span to div
Changed max-width from 500px to 480px
Changed margin-top from px to rem
